sonar-db/src/main/java/org/sonar/db/version/ClobColumnDef.java

```diff
@@ -42,10 +46,20 @@ public static Builder newClobColumnDefBuilder() {
 
   @Override
   public String generateSqlType(Dialect dialect) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "generateSqlType" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)